x-pack/test/security_solution_cypress/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -147,3 +156,33 @@ describe(
     });
   }
 );
+
+// this test suite doesn't run on serverless because it requires a custom role
+describe(
+  'Entity analytics management page - Risk Engine Privileges Callout',
```

Comment:
This one can be tested on serverless if you want. If you finally do so, please make sure you pass the test the experimental flag instead of having it on the global FTR configuration, note that all serverless tests are going to be executed soon on the second quality gate and we take the information from the test to setup the project.